transcom/mymove
# B-18459: Multi-Move Automated Regression Testing(Not Ready Just Testing)

[Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18459)
[Integration MR](https://github.com/transcom/mymove/pull/12482)

Summary
This story was to complete MultiMove automated playwright tests. Adjustments made in this MR includes getting rid of the skips on multi move tests. A MultiMove test is now only skipped if the Multi-Move feature flag is off. Some tests adjustments were made to match the Multi-Move workflow and attribute changes.

Just a few highlights that I have found:
Estimated incentive number expected in the test is different than what is actually in the header.
My thoughts: Did anything change with calculating the estimated incentive?

Solution: Check for incenative header to show up in th eplaywright test.

Pickup and desitination primary and secondary zipcodes error message handling seems to have changed.
Changes in this file were made about 2 months ago. Changes consisted of attribute name change but the error
My thoughts: I suspect the error messages are not working how the playwright tests expected because the component is not using the postalCodeValidator={validatePostalCode} prop that is being passed to it. The validatePostalCode() method which generates those error messages is being referenced but not being used.

Solution: These will be fixed in a future story to address the zipcode validation messages.

Setup to Run the Code
[Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
[Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
[Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)
How to test
Ensure Multimove feature flags are on. Check Circle CI to ensure playwright tests succeed.
